testnet/stacks-node/src/config.rs

```diff
@@ -1202,19 +1210,39 @@ impl FeeEstimationConfig {
         }
     }
 
-    pub fn make_scalar_fee_estimator<CM: CostMetric>(
+    pub fn make_scalar_fee_estimator<CM: 'static + CostMetric>(
         &self,
         mut chainstate_path: PathBuf,
         metric: CM,
-    ) -> ScalarFeeRateEstimator<CM> {
+    ) -> Box<dyn FeeEstimator> {
         if let Some(FeeEstimatorName::ScalarFeeRate) = self.fee_estimator.as_ref() {
             chainstate_path.push("fee_estimator_scalar_rate.sqlite");
-            ScalarFeeRateEstimator::open(&chainstate_path, metric)
-                .expect("Error opening fee estimator")
+            Box::new(
+                ScalarFeeRateEstimator::open(&chainstate_path, metric)
+                    .expect("Error opening fee estimator"),
+            )
         } else {
             panic!("BUG: Expected to configure a scalar fee estimator");
         }
     }
+
+    // Creates a fuzzed WeightedMedianFeeRateEstimator with window_size 5. The fuzz
+    // is uniform with bounds [+/- 0.5].
+    pub fn make_fuzzed_weighted_median_fee_estimator<CM: 'static + CostMetric>(
+        &self,
+        mut chainstate_path: PathBuf,
+        metric: CM,
+    ) -> Box<dyn FeeEstimator> {
+        if let Some(FeeEstimatorName::FuzzedWeightedMedianFeeRate) = self.fee_estimator.as_ref() {
+            chainstate_path.push("fee_fuzzed_weighted_median.sqlite");
+            let underlying_estimator =
+                WeightedMedianFeeRateEstimator::open(&chainstate_path, metric, 5)
+                    .expect("Error opening fee estimator");
+            Box::new(FeeRateFuzzer::new(underlying_estimator, 0.5))
```

Comment:
I think a tighter range than +/- 50% should be used for the fuzzer -- I think something as low as 10% would still work to be responsive during congestion.

Reply:
Better still, can this be a configuration variable, with default = 10%?

Reply:
Yes, we could make both fields configurable by adding two fields to the `FeeEstimationConfig`:

```
fee_rate_fuzzer_fraction: f64,
fee_rate_window_size: u64,
```

And these fields would only ever be used by the `WeightedMedianFeeRateEstimator` and `FeeRateFuzzer`.

In order for these to be configurable via the config file, the fields would also need to be added to the `FeeEstimationConfigFile` struct.

Reply:
I've added configuration options for this.

Reply:
marking as resolved because the discussion points have moved elsewhere